akka-docs/rst/java/distributed-pub-sub.rst

```diff
@@ -22,43 +22,60 @@ a few seconds. Changes are only performed in the own part of the registry and th
+same side of a network partition.
+
 You can send messages via the mediator on any node to registered actors on
-any other node. There are four modes of message delivery.
+any other node.
 
-**1. DistributedPubSubMediator.Send**
+There a two different modes of message delivery, explained in the sections
+:ref:`distributed-pub-sub-publish-java` and :ref:`distributed-pub-sub-send-java` below. 
 
-The message will be delivered to one recipient with a matching path, if any such
-exists in the registry. If several entries match the path the message will be sent
-via the supplied ``RoutingLogic`` (default random) to one destination. The sender() of the
-message can specify that local affinity is preferred, i.e. the message is sent to an actor
-in the same local actor system as the used mediator actor, if any such exists, otherwise
-route to any other matching entry. A typical usage of this mode is private chat to one
-other user in an instant messaging application. It can also be used for distributing
-tasks to registered workers, like a cluster aware router where the routees dynamically
-can register themselves.
+.. _distributed-pub-sub-publish-java:
 
-**2. DistributedPubSubMediator.SendToAll**
+Publish
+-------
 
-The message will be delivered to all recipients with a matching path. Actors with
-the same path, without address information, can be registered on different nodes.
-On each node there can only be one such actor, since the path is unique within one
-local actor system. Typical usage of this mode is to broadcast messages to all replicas
-with the same path, e.g. 3 actors on different nodes that all perform the same actions,
-for redundancy. You can also optionally specify a property (``allButSelf``) deciding
-if the message should be sent to a matching path on the self node or not.
+This is the true pub/sub mode. A typical usage of this mode is a chat room in an instant 
+messaging application.
 
-**3. DistributedPubSubMediator.Publish**
+Actors are registered to a named topic. This enables many subscribers on each node. 
+The message will be delivered to all subscribers of the topic. 
 
-Actors may be registered to a named topic instead of path. This enables many subscribers
-on each node. The message will be delivered to all subscribers of the topic. For
-efficiency the message is sent over the wire only once per node (that has a matching topic),
-and then delivered to all subscribers of the local topic representation. This is the
-true pub/sub mode. A typical usage of this mode is a chat room in an instant messaging
-application.
+For efficiency the message is sent over the wire only once per node (that has a matching topic),
+and then delivered to all subscribers of the local topic representation.
 
-**4. DistributedPubSubMediator.Publish with sendOneMessageToEachGroup**
+You register actors to the local mediator with ``DistributedPubSubMediator.Subscribe``. 
+Successful ``Subscribe`` and ``Unsubscribe`` is acknowledged with
```

Comment:
Should it say what does successful mean here? Has it managed to subscribe to the local mediator, or subscribed and got gossiped to other nodes etc.?